README.md

```diff
@@ -86,6 +86,22 @@ consumer.prometheus.metrics.reporter.listener=http://:8081
 consumer.auto.include.jmx.reporter=false
 ```
 
+### MirrorMaker
+
+The Source and Checkpoints MirrorMaker connectors expose [metrics](https://kafka.apache.org/documentation/#georeplication-monitoring). To collect these metrics with the reporter, add the metrics-reporter configurations to the connector configurations. Use the same listener (`prometheus.metrics.reporter.listener`) as Kafka Connect so that all metrics are exposed together on same endpoint.
+
+For example, if you configured Kafka Connect to export metrics with the listener set to `http://:8081`, do the same in the connector configuration:
+```json
+{
+    "name": "source",
+    "connector.class": "org.apache.kafka.connect.mirror.MirrorSourceConnector",
+    ...
+
+    "metric.reporters": "io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter",
+    "prometheus.metrics.reporter.listener": "http://:8081"
```

Comment:
Are you sure that we also need this config? I remember that it wasn't needed on my POC where I'm using 9404, but I may be wrong.

Reply:
You're right, it's not needed. All metrics are collected in the same registry so they are already exposed by the Connect listener anyway. I'll update that bit. While testing this I also found https://github.com/strimzi/metrics-reporter/issues/85

Reply:
One more thing: Why the different format here as opposed to properties elsewhere?

Reply:
Actually I think it's better to keep `"prometheus.metrics.reporter.listener": "http://:8081"`. While it's not needed, if it's not present, the reporter will also try to open port 8080. I think users will prefer only opening a single port.

Reply:
What about setting `"prometheus.metrics.reporter.listener.enable": "false"`? I tried and it looks good.

```sh
$ kubectl exec my-mm2-cluster-mirrormaker2-0 -- netstat -tulpn
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name    
tcp6       0      0 :::9404                 :::*                    LISTEN      129/java            
tcp6       0      0 :::8083                 :::*                    LISTEN      129/java            
tcp6       0      0 :::33953                :::*                    LISTEN      129/java
```

Reply:
I've updated the paragraph to include that